engine/gui/src/gui_script.cpp

```diff
@@ -551,6 +551,48 @@ namespace dmGui
         return 1;
     }
 
+    /*# gets the node type
+     *
+     * @name gui.get_type
+     * @param node [type:node] node from which to get the type
+     * @return type [type:constant] type
+     *
+     * - `gui.TYPE_BOX`
+     * - `gui.TYPE_TEXT`
+     * - `gui.TYPE_PIE`
+     * - `gui.TYPE_PARTICLEFX`
+     * - `gui.TYPE_CUSTOM`
+     * 
+     * @return subtype [type:hash|nil] name of custom type
```

Comment:
Regarding "hash" here, I think it's a bit confusing for the user, as they then might expect our Lua hash (the 64 bit hash user type).

I think it's better to say "[type: integer|nil] Id of the custom type"

Reply:
Okay, I think I understand. I'm not sure what the difference is yet between the different hash types yet. I referenced the custom node type from [this spine example](https://github.com/defold/extension-spine/blob/cab4430db95dec3fe3c89c7d5356f37f533d32eb/defold-spine/src/script_spine_gui.cpp#L25), which uses:

```c
static const uint32_t SPINE_NODE_TYPE = dmHashString32("Spine");
```

Is that not the same thing as the `hash: ["Spine"]` that shows up when you print a hash value in the editor?